### ISSUE TITLE
Vis generator and Stabilizer

### ISSUE THREAD

**Issue by Katapow99** [Azanor/thaumcraft-beta#1016]
so I was playing around with the mod and then came across the Stabilizer and thought it sounded really useful, I made it up and got a vis generator to power it, but I didn't have Immersive Engineering or any mods with power transfer quite yet so I just plugged the generator into the back of the stabilizer directly, then I got this crash report:
[crash-2018-06-14_14.10.15-server.txt](https://github.com/Azanor/thaumcraft-beta/files/2104058/crash-2018-06-14_14.10.15-server.txt)
I don't know if it's an issue of them being together or being run on a server, but I hope this can be fixed.

**Comment by Katapow99** [comment]
found out in a test world that it does the same thing if connected by wire so it must be a server issue.

**Comment by Dragnier** [comment]
Unless I'm mistaken, the vis generator and stabilizer do opposite things. The generator makes rf from vis and the stabilizer makes vis from rf. Putting them together sounds like an infinite loop to me.

**Comment by Sumguy62** [comment]
No. The stabilizer works on the infusion alter to reduce instability.

**Comment by Reassembly** [comment]
I did this (Stabilizer directly connected to Generator), and it worked/works just fine - but I'm not on a multiplayer server. From the crash dump, it kind of looks like an issue with the particle effects from the Stabilizer being improperly synchronized between server and client. So the good news is, your Stabilizer was (briefly) powered properly and working before the visual effects crashed the world. 😄 

@Dragnier  Not quite. The Stabilizer makes "stability" from RF, not vis. It's used to stabilize Infusion Matrices and rifts (and maybe other stuff in future).

**Comment by Dragnier** [comment]
@Reassembly Ah. Good to know and thanks for the clarification.

**Comment by Katapow99** [comment]
@Reassembly thanks but I know what was happening, I'm just reporting it so it can be replicated and fixed.

**Comment by Lukas45** [comment]
#969 Known and fixed
